_data/versions.yml

```diff
@@ -1186,6 +1186,48 @@ v3.2:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.2.0
 
 v3.1:
+- title: v3.1.7
+  components:
+    calico/node:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calico/releases/tag/v3.1.7
+    calicoctl:
+      version: v3.1.7
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.1.7
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v3.1.7/calicoctl
+    calico/cni:
+      version: v3.1.7
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.1.7
+      download_calico_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.7/calico
+      download_calico_ipam_url: https://github.com/projectcalico/cni-plugin/releases/download/v3.1.7/calico-ipam
+    calico/kube-controllers:
+      version: v3.1.7
+      url: https://github.com/projectcalico/k8s-policy/releases/tag/v3.1.7
+    felix:
+      version: v3.1.7
+      url: https://github.com/projectcalico/felix/releases/tag/3.1.7
+    networking-calico:
+      version: 3.1.3
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.1.3
```

Comment:
Not browse-able

Reply:
I'll fix this as pointed out here https://github.com/projectcalico/calico/pull/2802/files#r316377866